docs/topics/settings.rst

```diff
@@ -687,6 +687,34 @@ The Feed Temp dir allows you to set a custom folder to save crawler
 temporary files before uploading with :ref:`FTP feed storage <topics-feed-storage-ftp>` and
 :ref:`Amazon S3 <topics-feed-storage-s3>`.
 
+.. setting:: FTP_PASSIVE_MODE
+
+FTP_PASSIVE_MODE
+----------------
+
+Default: ``True``
+
+Whether or not to use passive mode when initiating FTP transfers.
+
+.. setting:: FTP_PASSWORD
+
+FTP_PASSWORD
+------------
+
+Default: ``"anonymous@example.com"``
```

Comment:
https://tools.ietf.org/html/rfc1635 says:

> Traditionally, this special anonymous user account accepts any string
   as a password, although it is common to use either the password
   "guest" or one's electronic mail (e-mail) address.  Some archive
   sites now explicitly ask for the user's e-mail address and will not
   allow login with the "guest" password.  Providing an e-mail address
   is a courtesy that allows archive site operators to get some idea of
   who is using their services.

Providing invalid email address will likely work in more cases, but using 'guest' as a default password instead of a fake email sounds more polite. +0 to change it to 'guest' (no strong opinion from me).